matching-service/index.js

```diff
@@ -9,12 +9,11 @@ const app = express();
 app.use(express.urlencoded({ extended: true }))
 app.use(express.json())
 app.use(cors()) // config cors so that front-end can use

```

Comment:
```suggestion
app.use(cors({
    origin: process.env.ENV === 'PROD'? process.env.FRONTEND_URL : 'http://localhost:3000',
}));
```

Reply:
Added, doesn't seem to affect it on live 👍